src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParser.cs

```diff
@@ -555,7 +558,7 @@ private RegexCharClass ScanCharClass(bool caseInsensitive, bool scanOnly)
                             if (!scanOnly)
                             {
                                 if (inRange)
-                                    throw MakeException(SR.Format(SR.BadClassInCharRange, ch.ToString()));
+                                    throw MakeException(RegexParseError.BadClassInCharRange, SR.Format(SR.BadClassInCharRange, ch.ToString()));
```

Comment:
Doesn't matter right now, but you could add an overload that takes char, and remove the ch.ToString()'s making slightly smalelr code